extensions/wikia/Recirculation/services/DiscussionsDataService.class.php

```diff
@@ -119,11 +119,12 @@ private function buildPost( $rawPost, $index ) {
 		$meta['authorAvatarUrl'] = $rawPost['createdBy']['avatarUrl'];
 		$meta['upvoteCount'] = $rawPost['upvoteCount'];
 		$meta['postCount'] = $rawPost['postCount'];
+		$meta['forumName'] = $rawPost['forumName'];
 
 		return new RecirculationContent( [
 			'url' => $this->server . '/d/p/' . $rawPost['id'],
 			'index' => $index,
-			'title' =>  wfShortenText( $rawPost['rawContent'], 120 ),
+			'title' =>  $rawPost['title'],
```

Comment:
Let's fallback to `wfShortenText( $rawPost['rawContent'], 120 )` - old posts don't have titles.